src/style/app.module.css

```diff
@@ -2178,6 +2205,21 @@ input[type='radio']:checked + label:hover {
 .listBoxOrgList .itemCardOrgList {
   width: 50%;
 }
+
+.createorgdropdown {
+  background-color: var(--grey-light);
+  border-color: var(--brown-color);
```

Comment:
Use variable names that describe the function of the color and not the color itself. This will help with future attempts to having Dark Mode.
- For example `--search-button-bg` is much better than `--light-blue`